runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/ControlClientPool.java

```diff
@@ -17,16 +17,12 @@
  */
 package org.apache.beam.runners.fnexecution.control;
 
-import org.apache.beam.sdk.fn.function.ThrowingConsumer;
-import org.apache.beam.sdk.util.ThrowingSupplier;
-
 /** Control client pool that exposes a source and sink of control clients. */
```

Comment:
<!--thread_id:cc_182896329_t; commit:1fd121da1417624b3b84f0300648251da64b9cb5; resolved:1-->
<!--section:context-quote-->
> **jkff** wrote:
> This terminology ("source" and "sink" in this context) is confusing. Could you clarify more about how this is used? It might also make sense to inline the "Source" and "Sink" interfaces as inner interfaces in this one (ControlClientPool.Source,Sink)

<!--section:body-->
Expanded. Let me know if this is any better.

I had them inlined previously but found this more readable due to most users only caring about their respective sources or sinks. If you have a strong preference for inner interfaces, I can change it; otherwise I'll leave it.

FYI, google-java-format did not like the multiline `@code` block that I added. I've used `<pre>` instead.